website/blog/2023-09-03-databend-weekly-109.md

```diff
@@ -0,0 +1,123 @@
+date: 2023-09-03
+slug: 2023-09-03-databend-weekly
+cover_url: 'weekly/weekly-109.jpg'
+image: 'weekly/weekly-109.jpg'
+tags: [weekly]
+description: "Stay up to date with the latest weekly developments on Databend!"
+contributors:
+  - name: akoshchiy
+  - name: andylokandy
+  - name: ariesdevil
+  - name: b41sh
+  - name: BohuTANG
+  - name: Chasen-Zhang
+  - name: dantengsky
+  - name: Dousir9
+  - name: drmingdrmer
+  - name: flaneur2020
+  - name: JackTan25
+  - name: lichuang
+  - name: PsiACE
+  - name: RinChanNOWWW
+  - name: SkyFan2002
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: wubx
+  - name: xudong963
+  - name: youngsofun
+  - name: zhang2014
+  - name: zhyass
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Understanding Cluster Key
+
+By defining a Cluster Key, you can enhance query performance by clustering tables. In this case, data will be organized and grouped based on the Cluster Key, rather than relying solely on the order in which data is ingested. This optimizes the data retrieval logic for large tables and accelerates queries.
+
+When using the `COPY INTO` or `REPLACE INTO` command to write data into a table that includes a cluster key, Databend will automatically initiate a re-clustering process, as well as a segment and block compact process.
+
+Clustering or re-clustering a table takes time. Databend suggests defining cluster keys primarily for sizable tables that experience slow query performance.
+
+If you are interested in learning more, please check out the resources listed below.
+
+- [Docs | Understanding Cluster Key](https://databend.rs/doc/sql-commands/ddl/clusterkey/)
+- [Docs | Databend Data Storage: Snapshot, Segment, and Block](https://databend.rs/doc/sql-commands/ddl/clusterkey/)
+
+## Code Corner
+
+Discover some fascinating code snippets or projects that showcase our work or learning journey.
+
+### Exploring Databend Local Mode
+
+**Databend Local Mode** aims to provide a simple Databend environment where users can directly interact with Databend using SQL without the need to deploy Databend services. This makes it convenient for developers to perform basic data processing using Databend.
+
+In local mode, a temporary `databend-query` process is started, which integrates both the client and server support. The storage is located in a temporary directory and its lifecycle follows the process. Once the process ends, the resources are also destroyed. You can start multiple local processes on one server, and their resources are isolated from each other.
```

Comment:
databend-local starts a temporary `databend-query` process which functions as both meta and query services.

Reply:
both client and server.